### PR TITLE
Add screen `target` address

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

**Comment by juliangruber** [comment]
@bajtos hindsight please

File: test.js [hunk 1]
```diff
@@ -189,4 +189,48 @@ describe('Withdraw', () => {
...
+    const withdraw = signer.address
+    const target = '0x1da5821544e25c636c1417ba96ade4cf6d2f9b5a' // sanctioned
+    const value = ethers.utils.parseUnits('1')
+
+    const digest = ethers.utils.solidityKeccak256(
+      ['address', 'uint256', 'address', 'address', 'uint256'],
+      [account, nonce, withdraw, target, value]
+    )
+    const signed = await signer.signMessage(digest)
+    const { v, r, s } = ethers.utils.splitSignature(signed)
```
Comment (bajtos) [inline test.js:217]: This is a lot of boilerplate code unrelated to the actual test. I would extract it into a helper method.

```js
const withdrawalRequest = createWithdrawalRequest({
   target: '0x1da5821544e25c636c1417ba96ade4cf6d2f9b5a' // sanctioned
});
// ...
const res = await fetch(`http://127.0.0.1:${server.address().port}`, {
 method: 'POST',
 body: JSON.stringify(withdrawalRequest)
})

**Review by bajtos — COMMENTED** [review]
LGTM 👍🏻

File: test.js (hunk 1 above)
Comment (juliangruber) [reply test.js:217]: This service isn't going to be used any more. I'll archive the repo

**Pull request merged**